test/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.Tests/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.Tests.fsproj

```diff
@@ -6,11 +6,21 @@
     <Compile Include="FSharpProjectTests.fs" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Include="TestData/*/*.fsproj;TestData/**/*.fs">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
+  <Target Name="RestoreTestProjects" BeforeTargets="Build">
```

Comment:
I'm not an MSBuild expert so there may be a better way of doing a restore then copy but I could only get this working reliably via two targets.